packages/web-client/tests/unit/models/workflow/workflow-session-test.ts

```diff
@@ -42,12 +42,13 @@ module('Unit | WorkflowSession model', function (hooks) {
   test('get string value', function (assert) {
     let workflowPersistence = new WorkflowPersistence();
     workflowPersistence.persistData(ID, {
-      name: 'EXAMPLE',
+      name: 'PREPAID_CARD_ISSUANCE',
```

Comment:
weird, this is one of the first things I tried, thinking that it was because it wasn’t a known `name`. I guess I did something else wrong!

Reply:
The change that matters is setting the `name` property on the object that we pass to the `WorkflowSession` constructor, this is just changed to be consistent/less confusing.